qutebrowser/commands/runners.py

```diff
@@ -172,6 +172,20 @@ def parse(self, text, *, aliases=True, fallback=False, keep=False):
                 return self.parse(new_cmd, aliases=False, fallback=fallback,
                                   keep=keep)
         try:
+            
```

Comment:
Is there a reason this is all inside the `try:`-block? I think it'd make more sense to only catch `KeyError` in the `cmd = cmdutils.cmd_dict[cmdstr]` line, but I might be missing something.